DNN-3796 Update RewriterUtils.cs

mortezakermani opened the pull request:
This is a fix for unicode character in URL issue reported in page below:
https://dnntracker.atlassian.net/browse/DNN-3796

jbrinkman commented:
When submitting pull requests, if you just update the branch with additional commits, it will automatically update the pull request.  This alleviates the need to submit multiple pull requests.  I'll close this one since #301 is a more up to date PR.

mortezakermani commented:
This code actually works for me:

//Encode querystring values to support unicode characters by M.Kermani
                queryString = HttpUtility.UrlDecode(queryString);
                System.Collections.Generic.List<string> listParameters = new System.Collections.Generic.List<string>();
                string text = "";
                string[] parameters = queryString.Split('&');
                for (int counter = 0; counter < parameters.Length; counter++)
                {
                    if (parameters[counter].Contains("="))
                    {
                        string[] arrparameter = parameters[counter].Split('=');
                        listParameters.Add(arrparameter[0] + "=" + HttpUtility.UrlEncode(arrparameter[1]));
                    }
                    else
                    {
                        text = "&" + parameters[counter];
                    }
                }
                queryString = String.Join("&", listParameters.ToArray()) + text;
